addons/binding/org.openhab.binding.gpstracker/src/main/java/org/openhab/binding/gpstracker/internal/handler/TrackerHandler.java

```diff
@@ -105,10 +125,44 @@ public String getTrackerId() {
 
     @Override
     public void initialize() {
+        if (sysLocation != null) {
+            createBasicDistanceChannel();
+        } else {
+            logger.debug("System location is not set. Skipping system distance channel setup.");
+        }
+
         mapDistanceChannels();
         updateStatus(ThingStatus.ONLINE);
     }
 
+    /**
+     * Create distance channel for measuring the distance between the tracker and the szstem.
+     */
+    private void createBasicDistanceChannel() {
+        Iterator it = thing.getChannels().stream().filter(c -> CHANNEL_TYPE_DISTANCE.equals(c.getChannelTypeUID())).iterator();
+        @Nullable ThingHandlerCallback callback = getCallback();
+        if (!it.hasNext() && callback != null) {
```

Comment:
This check does not make much sense - you are entering this if branch whenever there are distance channels defined, but independently of whether there is the system distance channel.

Reply:
The logic here was that tracker handlers are created without any distance channel. So the first time a tracker handler is activated it checks for distance channels and creates the first one for the system location. This applies to the previous comment as well.

Reply:
This is imho a false assumption. If a Tracker thing is defined in a things-file, the user can have distance channels defined in there, which thus are available on the Thing from the very first moment on.

Reply:
True. I think filtering with `homeDistanceChannelUID` resolves this.